include/defines.h

```diff
@@ -95,4 +95,16 @@
 #define TEXT_BLUE_GREEN_RED_CYCLE_1 4
 #define TEXT_BLUE_GREEN_RED_CYCLE_2 5
 
+
+/**
+ * Sound mode options
+ * Option 2 appears to be unused, as such its probably not
+ * a valid option
+**/
+#define SOUND_STEREO      0
+#define SOUND_HEADPHONES  1
+#define SOUND_UNUSED      2
+#define SOUND_MONO        3
+#define NUM_SOUND_OPTIONS 4
```

Comment:
You use soundMode but than  soundOptions here...

I think a better name for soundMode might exist. I'm sure there's a technical term for this kind of sound. Like soundSpeakerType or soundSetupOptions or something along those lines. This way people would immediately know what gSoundMode is.

Reply:
I'm open to suggestions, names are not my strong suit.
soundSpeakerType is my favorite of what you suggested